### PR TITLE
Fix the Uiviewer scaling issue

### PR DESCRIPTION
**Pull request by shaobinspire**

### CODE REVIEW

**Review by spireken — APPROVED** [review]

File: Applications/Spire/Include/Spire/Spire/Dimensions.hpp [hunk 1]
```diff
@@ -52,6 +52,35 @@ namespace Spire {
     \return The translated point.
   */
   QPoint translate(const QPoint& p) noexcept;
+
+  //! Scales the pixel width at the system's current DPI to 96 DPI.
```
Comment (kranar) [inline Applications/Spire/Include/Spire/Spire/Dimensions.hpp:56]: Fix the documentation.

**Review by kranar — CHANGES_REQUESTED** [review]

File: Applications/Spire/Include/Spire/Spire/Dimensions.hpp (hunk 1 above)
Comment (shaobinspire) [reply Applications/Spire/Include/Spire/Spire/Dimensions.hpp:56]: Do you mean to replace the word scales to the word unscales?

Comment (kranar) [reply Applications/Spire/Include/Spire/Spire/Dimensions.hpp:56]: More than just that, the documentation is incorrect, the function doesn't do what the documentation says it does.

**Review by kranar — APPROVED** [review]
The documentation is fine as is.

**Pull request merged**